src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
For my feeling I am in line with @ywwg and @rryan in this point. Maybe it is a issue of wording. 

For me "Headroom" is the just distance from a high value sample to 1.0. 

If the source track has already samples with the value 1.0 it uses the full available dynamic range, but can't be equalized or mixed without clipping.  Recent tracks are often normalized like that because it offers the best noise signal ratio. 

The confusion starts when we talking about the reasons for a headroom. 
* Dynamic Headroom: This allow tracks with different dynamics range (distance from the average sample value to the peak sample) played at equal loudness. 
* Mixing Headroom: If we add samples of two tracks they might exceed if we hit two peak samples. 

For my feeling we have a common sense that this PR and the Normalization Preferences only focus the "Dynamic Headroom"

The Mixing Headroom can be already tweaked separate with the Master Gain. 

When I understand @rryan and @ywwg concerns correct, it is only about the new "Default Boost" that is applied when no ReplayGain is applied. 
With a ReplayGain value in place the ReplayGain is responsible to center the Gain knob. If there is no ReplayGain value available or the feature is disabled, there is currently no option to tweak the pregain knob center for a Dynamic headroom. The  "Default Boost" fixes that. 
@rryan thinks this is not required. I think it is. It helps to predict the ReplayGain value during a running analyses and it helps to equalize the tracks manually using EQ and pregain knobs without being afraid the next track has more dynamic range and will clip with the current loudness.